tests/integration/users/test_viewset.py

```diff
@@ -165,9 +165,12 @@ def test_reset(self):
         user = User.objects.create(username='Test name ooMoa4ou4mohn2eo1ree')
         user.default_password = 'new_password_Yuuh8OoQueePahngohy3'
         user.save()
-        response = admin_client.post(reverse('user-reset-password', args=[user.pk]))
+        response = admin_client.post(
+            reverse('user-reset-password', args=[user.pk]),
+            {'password': 'new_password_Yuuh8OoQueePahngohy3'})
         self.assertEqual(response.status_code, status.HTTP_200_OK)
-        self.assertTrue(User.objects.get(pk=user.pk).check_password('new_password_Yuuh8OoQueePahngohy3'))
+        self.assertTrue(User.objects.get(pk=user.pk).check_password(
+            'new_password_Yuuh8OoQueePahngohy3'))
```

Comment:
Proposal: Add a new test that the password is reset even if no new password is send.